### PR TITLE
Move SubmissionCommentsTests to SwiftUITests

### PR DESCRIPTION
**Pull request by bootstraponline**
All tests passing!

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add a release note. If no release note is wanted, use `none`. Example: `release note: Fixed a bug that prevented users from enjoying the app.`
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add which apps this change affects. Example: `affects: Teacher, Student` or `affects: none`
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add a reference to a jira ticket. For example: `refs: MBL-10023`
  </td>
    </tr>
  </tbody>
</table>



Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 93.3% | 93.31% | -0.01%
**Student** | 92.27% | 92.27% | 0%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.65% | 92.65% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 93.31% | 93.31% | 0%
**Student** | 92.27% | 92.27% | 0%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.65% | 92.65% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Student/StudentUITests/Element/AudioRecorder.swift [hunk 1]
```diff
@@ -15,9 +15,25 @@
...
+    static var clearButton = app.find(AudioRecorder.clearButton)
+    static var recordButton = app.find(AudioRecorder.recordButton)
+    static var sendButton = app.find(AudioRecorder.sendButton)
+    static var stopButton = app.find(AudioRecorder.stopButton)
+    static var timeLabel = app.find(AudioRecorder.timeLabel)
+    static var currentTimeLabel = app.find(AudioRecorder.currentTimeLabel)
+    static var loadingView = app.find(AudioRecorder.loadingView)
+    static var playPauseButton = app.find(AudioRecorder.playPauseButton)
+    static var remainingTimeLabel = app.find(AudioRecorder.remainingTimeLabel)
+    static var timeSlider = app.find(AudioRecorder.timeSlider)
```
Comment (vanwagonet) [inline Student/StudentUITests/Element/AudioRecorder.swift]: if these are lazy initialized in a test, would other tests that use them end up referencing the wrong XCTestCase?

Comment (bootstraponline) [reply Student/StudentUITests/Element/AudioRecorder.swift]: Thanks. I pushed a fix to always recompute the app variable.

**Comment by bootstraponline** [comment]
PDF tests seem broken, also getting a weird mocking error.

image
image

7 tests are broken using the EarlGrey driver, I'm looking into that.

image

File: Student/StudentUITests/Submissions/SubmissionComments/SubmissionComments.swift [hunk 2]
```diff
@@ -15,43 +15,40 @@
...
-
-    static let addCommentButton = SubmissionComments(rawValue: "addCommentButton")
-    static let addMediaButton = SubmissionComments(rawValue: "addMediaButton")
-    static let commentTextView = SubmissionComments(rawValue: "commentTextView")
+    private static let className = String(describing: SubmissionComments.self)
 
-    static func attemptCell(submissionID: String, attempt: Int) -> SubmissionComments {
-        return SubmissionComments(rawValue: "attemptCell.submission-\(submissionID)-\(attempt)")
+    static func attemptCell(submissionID: String, attempt: Int) -> Element {
+        return app.find(id: "\(className).attemptCell.submission-\(submissionID)-\(attempt)")
```
Comment (vanwagonet) [inline Student/StudentUITests/Submissions/SubmissionComments/SubmissionComments.swift]: I think this is a good way to go forward.

**Pull request merged**